services/httpd/service.go

```diff
@@ -39,13 +44,35 @@ func NewService(c Config) *Service {
 // Open starts the service
 func (s *Service) Open() error {
 	// Open listener.
-	ln, err := net.Listen("tcp", s.addr)
-	if err != nil {
-		return err
+	if s.https {
+		cert, err := tls.LoadX509KeyPair(s.cert, s.cert)
+		if err != nil {
+			s.Logger.Printf("Failed to load HTTPS certificate '%s': %s", s.cert, err)
+			s.Logger.Println("Reverting to HTTP")
```

Comment:
I think it would be better to just return a `fmt.Sprintf("failed to load HTTPS certicated....")` error instead falling back to HTTP.